src/pages/AddPatient.jsx

```diff
@@ -5,237 +5,303 @@ import {
-                  required
-                />
-              </Grid>
-              <Grid size={3} item>
-                Mobile No
-                <TextField
-                  label="Phone Number"
-                  type="number"
-                  size="small"
-                  fullWidth
-                  required
-                />
-              </Grid>
-              <Grid size={3} item>
-                Address (Optional)
-                <TextField
-                  label="Apt/Suit/H No. Street"
-                  type="text"
-                  size="small"
-                  fullWidth
-                />
-              </Grid>
-              <Grid size={3} item>
-                City(Optional)
-                <TextField
-                  label="City"
-                  placeholder="City"
-                  type="text"
-                  size="small"
-                  fullWidth
-                />
-              </Grid>
-              <Grid size={3} item>
-                Zip code(Optional)
-                <TextField
-                  label="Zip"
-                  placeholder="Zip"
-                  type="text"
-                  size="small"
-                  fullWidth
-                />
-              </Grid>
-              <Grid size={3} item>
-                State(Optional)
-                <TextField
-                  label="State"
-                  placeholder="State"
-                  type="text"
-                  size="small"
-                  fullWidth
+      <>
+        <Card style={{ maxWidth: "65%", margin: "10% auto" }}>
+          <Grid container bgcolor="#1976d2" color="#fff" padding={2}>
+            <Grid size={10}>
+              <Typography variant="h5" align="center" marginLeft={12}>
+                Add Patient
+              </Typography>
+            </Grid>
+            <Grid size={2}>
+              <Typography align="right">
+                <Close
+                  onClick={onClose}
+                  sx={{
+                    boxShadow: 3,
+                    "&.hover": { boxShadow: 8 },
+                    borderRadius: 28
+                  }}
                 />
-              </Grid>
-              <Grid size={12} item align="center" marginTop={1}>
-                <Button
-                  type="reset"
-                  variant="contained"
-                  size="small"
-                  color="inherit"
-                >
-                  Cancel
-                </Button>
-                &nbsp;
-                <Button
-                  type="submit"
-                  variant="contained"
-                  size="small"
-                  color="success"
-                >
-                  Add
-                </Button>
-              </Grid>
+              </Typography>
             </Grid>
-          </form>
-        </CardContent>
-      </Card>
+          </Grid>
+          <CardContent>
+            <form onSubmit={handleSubmit(onSubmit)}>
```

Comment:
Can you just call on submit directly here?

Reply:
No . <form onSubmit={handleSubmit(onSubmit)}> works for react-hook-form